docs/usage/config-overview.md

```diff
@@ -108,6 +108,8 @@ Read the [Self-hosted experimental environment variables](./self-hosted-experime
 Finally, there are some special environment variables that are loaded _before_ configuration parsing because they are used during logging initialization:
 
 - `LOG_CONTEXT`: a unique identifier used in each log message to track context
+- `LOG_FILE`: used to specify the log file path
+- `LOG_FILE_LEVEL`: log file logging level, defaults to `info`
```

Comment:
I prefer it falls back to debug like today

Reply:
should renovate `LOG_LEVLE` also default to `debug` then?

@RahulGautamSingh If we need different levels, then add an optional default value. `coerceString` can help 🤗

Reply:
>I prefer it falls back to debug like today
This is confusing, because the [docs](https://docs.renovatebot.com/config-overview/#logging-variables) say, we fall back to `info`.

Reply:
stdout logging should default to info, file logging default to debug